docs/en/configuration/source-plugins/neo4j.md

```diff
@@ -0,0 +1,133 @@
+| [query_count](#query_count-string)                                       | string | no       | -             |
+| [relationship_nodes_map](#relationship_nodes_map-string)                 | string | no       | -             |
+| [relationship_source_labels](#relationship_source_labels-string)         | string | Yes      | -             |
+| [relationship_target_labels](#relationship_target_labels-string)         | string | Yes      | -             |
+
+### result_table_name [string]
+
+result table name
+
+### authentication_type [string]
+
+authentication type
+
+### authentication_basic_username [string]
+
+username
+
+### authentication_basic_password [string]
+
+password
+
+### url [string]
+url 
+
+### query [string]
+
+Cypher query to read the data
+
+### labels [string]
+
+List of node labels separated by :. The first label will be the primary label
+
+### relationship [string]
+
+Name of a relationship
+
+### schema_flatten_limit [string]
+Number of records to be used to create the Schema (only if APOC are not installed)
+
+### schema_strategy [string]
+
+Strategy used by the connector in order to compute the Schema definition for the Dataset. Possibile values are string, sample. When string it coerces all the properties to String otherwise it will try to sample the Neo4j’s dataset.
+
+### pushdown_filters_enabled [string]
+
+Enable or disable the Push Down Filters support
+
+### pushdown_columns_enabled [string]
+
+Enable or disable the Push Down Column support
+
+### partitions [string]
+
+This defines the parallelization level while pulling data from Neo4j.
+
+Note: as more parallelization does not mean more performances so please tune wisely in according to your Neo4j installation.
+
+### query_count [string]
+
+Query count, used only in combination with query option, it’s a query that returns a count field like the following:
+
+MATCH (p:Person)-[r:BOUGHT]->(pr:Product)
+WHERE pr.name = 'An Awesome Product'
+RETURN count(p) AS count
+or a simple number that represents the amount of records returned by query. Consider that the number passed by this value represent the volume of the data pulled of Neo4j, so please use it carefully.
+
+### relationship_nodes_map [string]
+
+
+If true return source and target nodes as Map<String, String>, otherwise we flatten the properties by returning every single node property as column prefixed by source or target
+
+### relationship_source_labels [string]
+
+List of source node Labels separated by :
+
+### relationship_target_labels [string]
+
+List of target node Labels separated by :
+
+## Example
+
+```bash
+   Neo4j {
+      result_table_name = "test"
+      authentication_type = "basic"
+      authentication_basic_username = "test"
+      authentication_basic_password = "test"
+      url = "bolt://localhost:7687"
+      labels = "Person"
+      #query = "MATCH (n1)-[r]->(n2) RETURN r, n1, n2 "
+   }
```

Comment:
This example does not match the required options described in `Options` above

Reply:
it was match yes* means you can choose one of it, not all yes* must configured